src/software/new_geom/cubic_bezier_spline2d.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include "software/new_geom/bezier_curve2d.h"
+#include "software/new_geom/ray.h"
```

Comment:
`Ray` seems unused? Can probably be removed from here and the `BUILD` deps

Reply:
ah yeah, that was carried over from an old implementation, removed here and in the `BUILD` file